run_audio_check.sh

```
@@ -1,6 +1,6 @@
 #!/bin/bash
 
 # Script para verificar e disparar notificações de áudio
-cd "/home/ball/Área de trabalho/BALL_VIGILANT_PRODUCAO_LINUX"
+cd "/Users/feliperesenderosa/Desktop/BallVigilantLinux"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)